src/fsharp/vs/QuickParse.fsi

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.FSharp.Compiler
+
+open System
+open Microsoft.FSharp.Compiler.SourceCodeServices
+
+#if COMPILER_PUBLIC_API
+type PartialLongName =
+#else
+type internal PartialLongName =
+#endif
+    { QualifyingIdents: string list
+      PartialIdent: string
+      LastDotPos: int option }
+    static member Default : PartialLongName
+
+#if COMPILER_PUBLIC_API
+module QuickParse =
+#else
+module internal QuickParse =
+#endif
+    val MagicalAdjustmentConstant : int
+    val CorrectIdentifierToken : s: string -> tokenTag: int -> int
```

Comment:
Minor things
* Could you give better argument names please, e.g. ``tokenText`` not ``s``?  
* Also in the implementation.  
* Also a `///` comment on each public thing please
* Add a blank line between before each ``///`` comment section